SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1793,3 +1796,107 @@ void USpatialNetDriver::DelayedSendDeleteEntityRequest(Worker_EntityId EntityId,
 		Sender->SendDeleteEntityRequest(EntityId);
 	}, Delay, false);
 }
+
+void USpatialNetDriver::HandleStartupOpQueueing(const TArray<Worker_OpList*>& InOpLists)
+{
+	if (InOpLists.Num() == 0)
+	{
+		return;
+	}
+
+	QueuedStartupOpLists.Append(InOpLists);
+	bQueueOpsUntilReady = FindAndDispatchStartupOps(InOpLists);
+
+	// We've found and dispatched all ops we need for startup, trigger BeginPlay()
+	// on the GSM and process the queued ops.  Note that FindAndDispatchStartupOps()
+	// will have notified the Dispatcher to skip the startup ops that we've
+	// processed already.
+	if (!bQueueOpsUntilReady)
```

Comment:
```suggestion
	if (bQueueOpsUntilReady)
	{
	    return;
	}
```
Then everything else can be indented inwards.